File: maria018/programação_basica/questão14.py
print(" >> Tempo de trabalho sem acidente <<")
dias = int(input(" Digite o tempo sem acidentes (em dias):"))
dias_copia = dias 
anos = dias// 365
dias = dias - (anos*356)
meses = dias // 30
dias = dias - (meses*30)

print(str(dias_copia) + "dias = " + str(anos) + " anos," + str(meses) + "meses e" + str(dias) + "dias")

File: maria018/programação_basica/Questão14.java
import java.util.Scanner;
public class Questão14 {
        public static void main (String[] args){
    
        Scanner s;
        int dias;
        int dias_copia;
        int anos;
        int meses;
        s = new Scanner(System.in);

        System.out.println(" >> Tempo de trabalho sem acidente << ");
        System.out.print(" Digite o tempo sem acidentes (em dias)");
        dias = s.nextInt();
        
        dias_copia = dias;
        anos = dias / 365;
        dias = dias - (anos*365);
        meses = dias / 30;
        dias = dias - (meses*30);

        System.out.println((dias_copia) + " dias = " + anos + "anos, " + meses + "meses é " + dias + "dias");

    }
}
